frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@apollo/client": "^3.3.16",
     "@chakra-ui/icon": "^1.1.11",
-    "@chakra-ui/react": "^1.6.5",
+    "@chakra-ui/react": "^1.7.1",
```

Comment:
rip still getting problems. set my windows default app mode to dark,
![image](https://user-images.githubusercontent.com/15113249/142555314-b5c4b140-84bd-4564-8c21-fd11a05fe69c.png)